bundles/binding/org.openhab.binding.novelanheatpump/README.md

```diff
@@ -148,3 +148,17 @@ Setpoint item=HeatPump_heating_temperature minValue=-10 maxValue=10 step=0.5
 Switch item=HeatPump_warmwater_operation_mode  mappings=[0="Auto", 1="Zuheizer", 2="Party", 3="Ferien", 4="Aus"]
 Setpoint item=HeatPump_warmwater_temperature minValue=10 maxValue=65 step=1
 ```
+
+## Troubleshooting
+Having troubles integrating your heatpump? Try this..
+1. Connect your heatpumpt to the LAN and configure the heatpump webserver (see manual). Activate DHCP Client. WebServer Password can be kept default (999999). Reboot your heatpump (power off/on) in order that IP connection works. 
```

Comment:
I don't see much difference between this and the existing statement that the network should be set up properly:

````
Make sure your heat pump is connected to your network and the network settings are configured.
````


Additionally, these statements sound like they would not be universally true, right? If not, these statements should also be removed.

The reboot suggestion is good as a troubleshooting tip, though.